models/roles.js

```diff
@@ -5,14 +5,20 @@ exports.roles = (() => {
   ac.grant("student")
     .readOwn("profile")
     .updateOwn("profile")
-    .deleteOwn("profile");
+    .deleteOwn("profile")
+    .readAny("issue")
+    .createOwn("issue")
+    .updateOwn("issue")
+    .deleteOwn("issue")
+    .createAny("comment");
 
-  ac.grant("student_moderator").extend("student");
+  ac.grant("student_moderator").extend("student").deleteAny("issue");
```

Comment:
I think we shouldn't give this right to student moderator. Rather only marking the issue as `inappropriate`